tests/unit/widget/widget_classes.js

```diff
@@ -0,0 +1,137 @@
+(function( $ ) {
+
+module( "widget factory", {
+	teardown: function() {
+		if ( $.ui ) {
+			delete $.ui.testWidget;
+			delete $.fn.testWidget;
+		}
+	}
+});
+
+TestHelpers.testJshint( "widget" );
+
+$.widget( "ui.classesWidget", {
```

Comment:
You should define what you need as you need it inside the actual tests. Most of this isn't actually needed in more than one test.